cookbooks/bcpc-hadoop/attributes/kerberos.rb

```diff
@@ -127,7 +127,16 @@
     princhost: '_HOST',
     perms: '0440',
     spnego_keytab: 'spnego.service.keytab'
-  }
+  },
+  ambari: {
```

Comment:
Please see the [HDFS-DU work](https://github.com/bloomberg/chef-bach/pull/932/files#diff-dea32ef5d89946f76672ef3d6b565138) for how to include this using a wrapper recipe to make Ambari a component in BCPC-Hadoop rather than a base feature.